app/components/dashboard-item/component.js

```diff
@@ -0,0 +1,31 @@
+import { computed } from '@ember/object';
+import moment from 'moment';
+import Component from '@ember/component';
+
+export default Component.extend({
+    ancestry: computed('node', 'node.{parent,root}', function() {
+        const rootId = this.get('node.root.id');
+        const parentId = this.get('node.parent.id');
+        const grandpaLink = this.get('node.parent.links.relationships.parent.links.related.href');
+        const grandpaId = grandpaLink ? grandpaLink.split('nodes')[1].split('/').filter(e => e)[0] : undefined;
```

Comment:
.filter(e => e) doesnt do anything :(. But, instead of making additional requests, will change this up so that grandpaLink is used to compare to rootLink, that way we don't need to mess with ids/